content/design/line-styles/index.html

```diff
@@ -2,65 +2,136 @@
-    code: '@include bb-border(tile)'
-    example: '<div class="bb-subsection-heading" style="border-bottom: 4px solid #e7eaec;">Tile crown</div>'
-    styling: '4px solid #e7eaec'
+<h2 class="bb-section-heading" id="generic-lines">Line styles</h2>
+  
+<div class="definition-table definition-table-wide">
+    <div class="definition-item">
+      <div class="bb-subsection-heading" style="border-bottom: 1px solid #e2e3e4;">Light separator</div>
+    </div>
+    <div class="definition-props">
+      <div class="definition-usage">
+        <div class="bb-field-label">Usage</div>
+        <div>The light separator is a very light gray, single-pixel line that subdivides content within a container. Do not use it at the edge of a container. Only use it when it is adjacent to white space on both sides.</div>
+        <p>SKY UX uses this style for:</p>
+        <ul>
+          <li>Subsection dividers within containers such as <a routerLink="../../components/tiles">tiles</a></li>
+          <li>Top boundaries for action bars on <a routerLink="../../components/modal">modals</a></li>
+        </ul>
+      </div>
+      <div class="definition-code">
+        <div class="bb-field-label">Code</div>
+        <code>@include bb-border(light)</code>
+      </div>
+      <div class="definition-styling">
+        <div class="bb-field-label">Styling</div>
+        <ul class="list-unstyled">
+          <li>1px solid #e2e3e4</li>
+          <li>When this line style subdivides content within a container, use a 15px margin on both ends.</li>
+        </ul>
+      </div>
+    </div>
+  </div>
 
-  - usage: This line separates members of a homogenous set. It separates the rows in <a href="../../components/grids">grids</a> or <a href="../../components/repeater">repeaters</a>. The dotted style lowers its visual weight, and the moderate gray allows it to work with or without row striping on the list rows. It stretches to the full width of the rows (not the width of the container that holds the rows).
-    code: '@include bb-border(row)'
-    example: '<div class="bb-subsection-heading" style="border-bottom: 1px dotted #ccc;">Row separator</div>'
-    styling: '1px dotted #ccc'
+  <div class="definition-table definition-table-wide">
+    <div class="definition-item">
+      <div class="bb-subsection-heading" style="border-bottom: 1px solid #cdcfd2;">Dark separator</div>
+    </div>
+    <div class="definition-props">
+      <div class="definition-usage">
+        <div class="bb-field-label">Usage</div>
+        <div>The dark separator is a moderate gray, single-pixel line. This heavier stroke holds up between two different colors, which makes it ideal for borders and boundaries. It also frequently separates active content from passive content. For example, it can separate a toolbar from the content that the toolbar acts on.</div>
+            <p>SKY UX uses this style for:</p>
```

Comment:
This line is missing space between it and the preceding paragraph. I think it would look better to just pull this text into the previous paragraph. If you prefer to keep it as a separate paragraph, I think we should add `p` tags around the preceding paragraph to give it the necessary spacing.